src/tokens/selectors.ts

```diff
@@ -131,6 +138,26 @@ export const defaultTokenToSendSelector = createSelector(
   }
 )
 
+export const staleTokenBalanceSelector = createSelector(
+  [tokensListSelector, tokensWithStaleUsdValueSelector, localCurrencyExchangeRatesSelector],
+  (tokensList, tokensWithUsdValue, exchangeRate) => {
```

Comment:
nit: rename `tokensWithUsdValue` to `tokensWithStaleUsdValue`

Reply:
lol good one, done